scripts/docker-cron.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
detail: the script does index update/index rotation, so a name like `index-rotation.sh` would be more descriptive than `docker-cron.sh`, even if the script is executed from a cron job